packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -23,6 +23,7 @@ const StyledTooltip = styled.div`
   /* Overriding the default popover styles */
   display: none;
   &[popover] {
+    position: absolute;
```

Comment:
AIUI this is the main thing that is doing all the heavy lifting. I think you could just apply this line and ignore the rest and it would still work?

Reply:
Yeah definitely this line does all the heavy lifting and actually the rest is not _directly_ related to this change but the existing issue* just became more visible when we added this line (no idea why 🤷🏻‍♀️ ). 

*The existing issue (at least it is an issue in my opinion) is where we set the position. 

Previously the flow was:
1) hover/focus event is triggered
2) `tooltip.showPopover()` function is called
3) `:popover-open` attribute is added to the tooltip div
4) Adding `:popover-open` triggers the `toggle` event as well as triggers the animation that changes the tooltip's opacity from 0 to 1
5) Position is set in the `toggle` event's callback function.

I believe the wiggling issue (video in the PR description) happens because we set the position a little too late (step 5) but we make the tooltip visible in the step 3. So in this PR I am setting the position in Step 3 and that makes the toggle event redundant. 

Let me know if you have any concern or suggestions.